lib/creditsafe/match_type.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+module Creditsafe
+  module MatchType
+    MATCH_BEGINNING = "MatchBeginning"
+    MATCH_BLOCK = "MatchBlock"
+    EXACT_VALUE = "ExactValue"
+    MATCH_WORDS = "MatchWords"
+    CLOSEST_KEYWORDS = "ClosestKeywords"
+    MATCH_BLOCK_OR_WORDS = "MatchBlockOrWords"
+    MATCH_ANY_PARTS = "MatchAnyParts"
+
+    ALLOWED = {
+      DE: [MATCH_BEGINNING, MATCH_BLOCK, EXACT_VALUE, MATCH_WORDS],
+      FR: [MATCH_BEGINNING, CLOSEST_KEYWORDS, EXACT_VALUE],
+      GB: [MATCH_BEGINNING, MATCH_BLOCK_OR_WORDS, EXACT_VALUE],
+      IE: [MATCH_BEGINNING, MATCH_BLOCK_OR_WORDS, MATCH_WORDS],
+      GL: [MATCH_BEGINNING],
+      CH: [CLOSEST_KEYWORDS],
+      LI: [CLOSEST_KEYWORDS],
+      NL: [MATCH_BEGINNING, MATCH_BLOCK_OR_WORDS, MATCH_WORDS],
+      CZ: [MATCH_BEGINNING, MATCH_BLOCK],
+      IS: [MATCH_BEGINNING],
+      LT: [MATCH_ANY_PARTS],
+      MT: [MATCH_BEGINNING, MATCH_BLOCK],
+      SK: [MATCH_BEGINNING, MATCH_BLOCK],
+      BE: [MATCH_BEGINNING, MATCH_BLOCK_OR_WORDS, MATCH_WORDS],
+      SE: [MATCH_BLOCK_OR_WORDS, CLOSEST_KEYWORDS],
+      PL: [CLOSEST_KEYWORDS],
+      NO: [CLOSEST_KEYWORDS],
+      PT: [MATCH_BEGINNING],
+      LU: [MATCH_BEGINNING, MATCH_BLOCK, EXACT_VALUE],
+      CA: [MATCH_BEGINNING],
+      ES: [CLOSEST_KEYWORDS],
+      US: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      PR: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      AS: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      FM: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      GU: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      MH: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      MP: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      PW: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      VI: [MATCH_BEGINNING, CLOSEST_KEYWORDS],
+      SI: [MATCH_BLOCK_OR_WORDS],
+      AL: [MATCH_BLOCK_OR_WORDS],
+      BA: [MATCH_BLOCK_OR_WORDS],
+      HR: [MATCH_BLOCK_OR_WORDS],
+      MK: [MATCH_BLOCK_OR_WORDS],
+      BG: [MATCH_BLOCK_OR_WORDS],
+      RO: [MATCH_BLOCK_OR_WORDS],
+      ME: [MATCH_BLOCK_OR_WORDS],
+      RS: [MATCH_BLOCK_OR_WORDS],
+      LV: [MATCH_BLOCK_OR_WORDS],
+      EE: [MATCH_BLOCK_OR_WORDS],
+      UA: [MATCH_BLOCK_OR_WORDS],
+      MD: [MATCH_BLOCK_OR_WORDS],
+      RU: [MATCH_BLOCK_OR_WORDS],
+      AM: [MATCH_BLOCK_OR_WORDS],
+      AZ: [MATCH_BLOCK_OR_WORDS],
+      BY: [MATCH_BLOCK_OR_WORDS],
+      GE: [MATCH_BLOCK_OR_WORDS],
+      KZ: [MATCH_BLOCK_OR_WORDS],
+      UZ: [MATCH_BLOCK_OR_WORDS],
+      TJ: [MATCH_BLOCK_OR_WORDS],
+      TM: [MATCH_BLOCK_OR_WORDS],
+      KG: [MATCH_BLOCK_OR_WORDS],
+      KM: [MATCH_BLOCK_OR_WORDS],
+      HK: [MATCH_BLOCK_OR_WORDS],
+      AT: [MATCH_BLOCK_OR_WORDS],
+      IT: [CLOSEST_KEYWORDS],
+      BT: [CLOSEST_KEYWORDS],
```

Comment:
I think this should be `BR` (I'm comparing with Creditsafe's spreadsheet)

Reply:
Yep!